install/wayscript/install.yml

```diff
@@ -0,0 +1,22 @@
+id: wayscript
+
+name: WayScript
+
+title: WayScript
+
+description: |
+  WayScript is an Internal Developer Platform (IDP) that enables scalable, single click development environments. Connect your environments to New Relic in minutes.
+
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: nerdlet
+  destination:
+    nerdletId: external-install.install 
+
+fallback:
+  mode: link
+  destination:
+    url: https://wsxdocs.wayscript.com/quickstart
```

Comment:
https://github.com/newrelic/newrelic-quickstarts/blob/main/data-sources/argocd/config.yml#L8-L17

Reply:
ignore this